lib/wptool.js

```diff
@@ -260,7 +260,7 @@ function nativeEnumerate(wpsdk, options, next) {
 			} else {
 				var emulators = parseAppDeployCmdListing(out, wpsdk);
 				next(null, {
-					devices: [{name: 'Device', udid: 0, index: 0, wpsdk: null}],
+					devices: [],
```

Comment:
Maybe so with newer versions of WP SDK, but the older WP SDK would always return a "device" entry regardless if there weren't any connected devices or if there were more than one.

The list of emulators also included the "device" entry and it was/is always present. I'm guessing we filter it out, but then were manually setting it for device.detect().

I will need to look into this further.